tests/System/expected/test_webContext__TagManager.getAvailableTagTypesInContext.xml

```diff
@@ -293,7 +293,7 @@
 						<uiControlAttributes>
 						</uiControlAttributes>
 						<availableValues />
-						<description>Your Emarsys Merchant ID</description>
+						<description>Your Emarsys Merchant ID.</description>
```

Comment:
Should we be adding `.` where they didn't already exist?

Reply:
I just added for consistency, Ill remove it so that it doesn't show a change